src/js/bootstrap-slider.js

```diff
@@ -991,14 +1007,18 @@ const windowIsDefined = (typeof window === "object");
 				this.handle1.removeEventListener("keydown", this.handle1Keydown, false);
 				this.handle2.removeEventListener("keydown", this.handle2Keydown, false);
 
-				//rmove the listeners from the ticks if they had their own listeners
-				if(this.options.ticks_tooltip){
-					var ticks = document.getElementsByClassName('slider-tick-container')[0].children;
+				//remove the listeners from the ticks and handles if they had their own listeners
+				if (this.options.ticks_tooltip) {
+					var ticks = this.ticksContainer.getElementsByClassName('slider-tick');
 					for(var i = 0; i < ticks.length; i++ ){
-						ticks[i].removeEventListener('mouseenter', this.createTickMouseOverListener, false);
-						ticks[i].removeEventListener('mouseleave', this.createTickMouseOverListener, false);
+						ticks[i].removeEventListener('mouseenter', this.ticksCallbackMap.mouseEnter, false);
```

Comment:
If you map the tick index to the event listener references as I mentioned above, you should be able to do the following here in order to cleanup the references:

```js
var tickEventListeners = this.ticksCallbackMap[i];
var tickElement = ticks[i];

tickElement.removeEventListener('mouseenter', tickEventListeners.mouseEnter, false);
tickElement.removeEventListener('mouseleave', tickEventListeners.mouseLeave, false);
```